common/mock-radio.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Mock radio for interfacing with mock devices
+ */
+
+export interface MockUDPListener {
+  onUDPMessage(msg: Buffer, rinfo: Object): void;
+}
+
+// TODO(cjdaly) Builders
+export class DiscoveryData {
+  id: string;
+  model: string;
+  hwRev: string;
+  fwRev: string;
+  channels: number[];
+  constructor(
+    id: string,
+    model: string,
+    hwRev: string,
+    fwRev: string,
+    channels: number[]
+  ) {
+    this.id = id;
+    this.model = model;
+    this.hwRev = hwRev;
+    this.fwRev = fwRev;
+    this.channels = channels;
+  }
+}
+
+// Simulates a network with simple UDP messaging functionality
+export class MockNetwork {
+  udpListeners: Map<string, MockUDPListener[]>;
+
+  constructor() {
+    this.udpListeners = new Map<string, MockUDPListener[]>();
+  }
+
+  registerUDPListener(
+    listener: MockUDPListener,
+    port: number,
+    address: string
+  ) {
+    const key: string = address + ':' + port.toString();
```

Comment:
I believe `string` is implied here and won't need to be defined explicitly.

Reply:
Okay, will remove.  I wasn't sure when I should explicitly add types.